partiql-types/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "partiql-types"
```

Comment:
Do you also need to add this crate to the workspace's `Cargo.toml`: https://github.com/partiql/partiql-lang-rust/blob/5d105335c2b35f7f9fd911db6ebd2ce810641eb3/Cargo.toml?

Reply:
Oh good catch, will add!